programmers/다트-게임.js

```diff
@@ -0,0 +1,57 @@
+function solution(dartResult) {
+    let num = 0;
+    const answer = [];
+    let temp = 0;
+    
+
+    for (let i = 0; i < dartResult.length; i++) {
+        if (dartResult[i] >= 0 && dartResult[i] <= 9) {
+            if (dartResult[i] == 1 && dartResult[i + 1] == 0) {
```

Comment:
```suggestion
            if (dartResult[i] === 1 && dartResult[i + 1] === 0) {
```
동일하다를 뜻할때는 javascript에서는 ===가 좀 더 확실한 뜻을 가집니다